src/components/HeaderAuth.jsx

```diff
@@ -50,10 +45,9 @@ HeaderAuth.defaultProps = {
   initialised: false,
 };
 
-function mapStateToProps(state) {  // Listens for changes in the Redux Store
-  return {
-    user: state.login.user,
-    initialised: state.login.initialised,
-  };
-}
+const mapStateToProps = (state) => ({
```

Comment:
It's a _very_ slightly more terse way of passing that object to the `connect` component, which is ultimately what it needs. I can change it back if you'd prefer?